src/main/java/core/basesyntax/strategy/OperationServiceStrategy.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.service.OperationService;
+import core.basesyntax.service.impl.AddOperationService;
+import core.basesyntax.service.impl.SubtractOperationService;
+
+public class OperationServiceStrategy {
+    public OperationService getOperationService(String operation) {
+        switch (operation) {
+            case "b" :
+            case "s" :
+            case "r" :
+                return new AddOperationService();
+            default:
+                return new SubtractOperationService();
+        }

```

Comment:
as described in common mistakes, let's avoid switch cases and use map with strategies in such cases